### ISSUE TITLE
Unable to build the module-federation example

### ISSUE THREAD

**Issue by parichit13** [callstack/repack#298]
## Environment
```
System
  platform           darwin                                                           
  arch               arm64                                                            
  cpu                10 cores          Apple M1 Pro                                   
  Node.js v18.9.1
  yarn v3.2.1
```

## Description
Unable to build module-federation example for production.

I have been able to run the module-federation example on dev but I have been trying to change the implementation to an actual production level code. So I started to build production bundles for each of the modules.

I have been able to produce release bundles for **host** and **app2** MFs using the following command:

`STANDALONE=1 webpack -c webpack.config.mjs --env platform=ios --env mode=production`

But when trying to run the same command on **app1** and **module1** MFs, I am getting the following error:

```
[webpack-cli] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
```

## Reproducible Demo

```
git clone https://github.com/callstack/repack-examples
cd repack-examples
yarn run bootstrap
cd app1
npx STANDALONE=1 webpack -c webpack.config.mjs --env platform=ios --env mode=production
```

**Output**

```
[webpack-cli] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (internal/errors.js:322:7)
    at validateString (internal/validators.js:124:11)
    at Object.join (path.js:1148:7)
    at AssetsCopyProcessor.enqueueChunk (/Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/@callstack/repack/dist/webpack/plugins/utils/AssetsCopyProcessor.js:53:70)
    at /Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/@callstack/repack/dist/webpack/plugins/OutputPlugin.js:225:55
    at Hook.eval [as callAsync] (eval at create (/Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:8:17)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/tapable/lib/Hook.js:18:14)
    at /Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/webpack/lib/Compiler.js:882:27
    at /Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/neo-async/async.js:2818:7
    at done (/Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/neo-async/async.js:3522:9)
    at alreadyWritten (/Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/webpack/lib/Compiler.js:714:8)
    at /Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/webpack/lib/Compiler.js:802:19
    at /Users/parichitkotawala/Development/RN/repack-examples/module-federation/module1/node_modules/graceful-fs/graceful-fs.js:123:16
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:71:3) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by jbroma** [comment]
This issue was caused by `OutputPlugin` trying to copy non-existing chunk. This issue was fixed in #347 

Updating the Re.Pack to the newest version should solve the problem.

**Comment by RafikiTiki** [comment]
Hey @parichit13 I added [a comment](https://github.com/callstack/repack-examples/issues/7#issuecomment-1618426371) in the original issue in the repack-examples repo. Closing this, as it's not an issue with Repack itself but rather an old version of Repack being used in the examples.
